core/ConVarManager.cpp

```diff
@@ -232,12 +236,26 @@ void ConVarManager::OnPluginUnloaded(IPlugin *plugin)
 
 void ConVarManager::OnClientDisconnected(int client)
 {
+	IPluginFunction *pCallback = NULL;
 	/* Remove convar queries for this client that haven't returned results yet */
 	for (List<ConVarQuery>::iterator iter = m_ConVarQueries.begin(); iter != m_ConVarQueries.end();)
 	{
 		ConVarQuery &query = (*iter);
 		if (query.client == client)
 		{
+			pCallback = query.pCallback;
+			if (pCallback)
+			{
+				cell_t ret;
+
+				pCallback->PushCell(InvalidQueryCvarCookie);
+				pCallback->PushCell(client);
+				pCallback->PushCell(eQueryCvarValueStatus_Cancelled);
+				pCallback->PushString("\0");
+				pCallback->PushString("\0");

```

Comment:
Any reason for the double-nul in these strings here?

```suggestion
				pCallback->PushString("");
				pCallback->PushString("");
```

Reply:
No particular reason, just thought it was a convention, I'll change it